docs/en/sdk-manual/web.md

```diff
@@ -70,167 +236,220 @@ Clickstream Web SDK supports the following data types:
-| _error_code              | The `_clickstream_error` event's attribute.                                                                                                                |
-| _page_title              | Added in all events.                                                                                                                                       |
-| _page_url                | Added in all events.                                                                                                                                       |
+### Common attributes
+
+| Attribute name  | Data type | Description                                                       | How to generate                                                                                                                                                                                                                                            | Usage and purpose                                                                                     |
+|-----------------|-----------|-------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------|
+| hashCode        | string    | the event object's hash code                                      | calculated by library `@aws-crypto/sha256-js`                                                                                                                                                                                                              | distinguish different events                                                                          |
+| app_id          | string    | clickstream app id                                                | generated when clickstream app create from solution web console.                                                                                                                                                                                           | identify the events for your apps                                                                     |
+| unique_id       | string    | the unique id for user                                            | generated from `uuidV4()` when the sdk first initialization<br> it will be changed after user re-login to another user who never login, and when user re-login to the before user in same browser, the unique_id will reset to the before user's unique_id | the unique for identity different users and associating the behavior of logging in and not logging in |
```

Comment:
- it will be changed after user re-login to another user who never login, and when user re-login to the before user in same browser, the unique_id will reset to the before user's unique_id --> it will be changed if user logout and then login to a new user. When user re-login to the previous user in the same browser, the unique_Id will be reset to the same previous unique_id.
- the unique for identity different users and associating the behavior of logging in and not logging in --> the unique id to identity different users and associating the behavior of logged-in and not logged-in.

Reply:
fixed.